### PR TITLE
CALCULATOR WEB APP

### PR DESCRIPTION
**Pull request by VISWANATH78**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

# Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by VISWANATH78** [comment]
Please Accept my first Calculator APP developed using HTML,CSS. might help new developers to understand how it works

File: hacktober/Calculator-App-master/Calculator-App-master/AUTHORS [hunk 1]
```diff
@@ -0,0 +1 @@
+Tsepas Athanasios - Full Stack Developer
```
Comment (anjali1102) [inline hacktober/Calculator-App-master/Calculator-App-master/AUTHORS:1]: add your own projects :\

Comment (VISWANATH78) [reply hacktober/Calculator-App-master/Calculator-App-master/AUTHORS:1]: Sorry uploaded wrong one

Comment (VISWANATH78) [reply hacktober/Calculator-App-master/Calculator-App-master/AUTHORS:1]: I do update the one i did

**Pull request closed without merging**